site/docs/reference/Connectors/materialization-connectors/google-cloud-sql-mysql.md

```diff
@@ -0,0 +1,327 @@
+| `/advanced/sslmode`         | SSL Mode               | Overrides SSL connection behavior by setting the 'sslmode' parameter.                                                                                                                                                                                                                                                                                                                      | string |                  |
+| `/advanced/ssl_server_ca`   | SSL Server CA          | Optional server certificate authority to use when connecting with custom SSL mode                                                                                                                                                                                                                                                                                                          | string |                  |
+| `/advanced/ssl_client_cert` | SSL Client Certificate | Optional client certificate to use when connecting with custom SSL mode.                                                                                                                                                                                                                                                                                                                   | string |                  |
+| `/advanced/ssl_client_key`  | SSL Client Key         | Optional client key to use when connecting with custom SSL mode.                                                                                                                                                                                                                                                                                                                           | string |                  |
+
+### Setting the MySQL time zone
+
+MySQL's [`time_zone` server system variable](https://dev.mysql.com/doc/refman/5.7/en/server-system-variables.html#sysvar_time_zone) is set to `SYSTEM` by default.
+
+If you intend to materialize collections including fields of with `format: date-time` or `format: time`,
+and `time_zone` is set to `SYSTEM`,
+Flow won't be able to detect the time zone and convert datetimes to the
+appropriate timezone when materializing.
+To avoid this, you must explicitly set the time zone for your database.
+
+You can:
+
+* Specify a numerical offset from UTC.
+   - For MySQL version 8.0.19 or higher, values from `-13:59` to `+14:00`, inclusive, are permitted.
+   - Prior to MySQL 8.0.19, values from `-12:59` to `+13:00`, inclusive, are permitted
+
+* Specify a named timezone in [IANA timezone format](https://www.iana.org/time-zones).
+
+* If you're using Amazon Aurora, create or modify the [DB cluster parameter group](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/USER_WorkingWithDBClusterParamGroups.html)
```

Comment:
Info on aurora should come out of this version

Reply:
done